index.js

```diff
@@ -40,6 +40,11 @@ import androidBgMessagingHandler from './src/core/messaging/silent/android-backg
 import { YellowBox } from 'react-native';
 YellowBox.ignoreWarnings(['RCTRootView cancelTouches', 'Require cycle']);
 
+// see https://github.com/facebook/react-native/issues/16434
+import { URL, URLSearchParams } from 'whatwg-url';
```

Comment:
What impact/problem does this have?

Reply:
default `URL` is not implemented on iOS, throws an error, so we need to change the base import of this
I think `URL` and `URLSearchParams` are quite useful to use, they'll help us a lot when parsing urls

@krisboit what do you think about this?

Reply:
I have switched to `string.replace`